diff -urN virtualjaguar/src/gui/mainwin.cpp virtualjaguar-patched/src/gui/mainwin.cpp
--- virtualjaguar/src/gui/mainwin.cpp	2012-01-06 09:18:10.000000000 +1100
+++ virtualjaguar-patched/src/gui/mainwin.cpp	2012-10-12 02:38:20.601047563 +1100
@@ -53,10 +53,7 @@
 #include "jagstub2bios.h"
 #include "joystick.h"
 
-#ifdef __GCCWIN32__
-// Apparently on win32, usleep() is not pulled in by the usual suspects.
 #include <unistd.h>
-#endif
 
 // The way BSNES controls things is by setting a timer with a zero
 // timeout, sleeping if not emulating anything. Seems there has to be a
